src/Common/src/System/Net/SecurityProtocol.cs

```diff
@@ -15,6 +15,8 @@ internal static class SecurityProtocol
         public const SslProtocols DefaultSecurityProtocols =
             SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
 
+        public const SslProtocols SystemDefaultSecurityProtocols = SslProtocols.None;
```

Comment:
>  It doesn't look like you're touching System.Net.Http as part of this change, but whoever updates WinHttpHandler to respect the new setting

Nothing can be done for WinHttp(Handler) with the existing Win32 API set. I have started the discussion with the WinHTTP team internally:

> I have confirmed that WinHTTP does not support this setting